src/components/BooksList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
+
+// end imports
+
+const BookList = () => {
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
+
+  const handleRemoveBook = (id) => {
+    dispatch(removeBook({ item_id: id }));
+  };
+
+  return (
+    <div>
+      <h2>Book List</h2>
+      <ul>
+        {books.map((book) => (
+          <li key={book.item_id}>
+            {book.title}
+            {' '}
+            by
+            {book.author}
+            <button type="button" onClick={() => handleRemoveBook(book.item_id)}>Remove Book</button>
+          </li>
```

Comment:
- **[OPTIONAL]** There should be a component for displaying one book, from a previous project. Kindly export this code to a new component and import it here instead. This BookList component should simply be for fetching the books from the store and using the map and the one-book component to display the books.